1.9/overview/architecture/node-types.md

```diff
@@ -0,0 +1,79 @@
+---
+post_title: Node Types
+nav_title: Node Types
+menu_order: 1
+---
+
+A DC/OS node is a virtual or physical machine on which DC/OS components run. DC/OS nodes are networked together to form a DC/OS cluster.
+
+A DC/OS cluster is composed of three types of nodes: masters, private agents, and public agents.
+
```

Comment:
Is DMZ common terminology or do we need to introduce, or spell out in the graphic?